frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -372,7 +373,7 @@ local function genAutoSaveMenuItem(value)
     if not value then
         text = _("Only on close, suspend and exit")
     else
-        text = T(_("Every %1 mn"), value)
+        text = T(N_("Every 1 minute", "Every %1 minutes", value), value)
```

Comment:
Did I write that? If so, I meant:

```suggestion
        text = T(N_("Every minute", "Every %1 minutes", value), value)
```

Reply:
Oh right, I wrote `Every 1 m`, which does work. ;-)